src/components/Header/index.tsx

```diff
@@ -33,40 +36,52 @@ export function Header({ theme, toggleTheme }: Props) {
 
     return (
         <header>
+            <>
```

Comment:
This React Fagment has no purpose. Please remove

Reply:
Actually this whole section you added is wrong. It breaks the position of hamburger menu button position.